src/domUpdates.js

```diff
@@ -1,80 +1,212 @@
 let domUpdates = {
   randomNumber :  Math.floor(Math.random() * 49) + 1,
 
-  grabRecipes() {
-    fetch('https://fe-apps.herokuapp.com/api/v1/whats-cookin/1911/recipes/recipeData')
-    .then(response => response.json())
-    .then(recipeObject => {
-      recipeObject.recipeData.forEach(recipe =>{
-       displayAllRecipes(recipe);
+  
+   displayAllRecipes(recipes,favorites) {
+     
+    if (recipeCards === null) {
+      return 
+    }
+    recipeCards.innerHTML = ''
+    recipes.forEach(recipe => {
+      let src = "https://image.flaticon.com/icons/svg/149/149222.svg"
+      if (favorites.includes(String(recipe.id))) {
+         src = "https://image.flaticon.com/icons/svg/148/148841.svg"
+      }
+    recipeCards.innerHTML += `<article id = ${recipe.id} class="single-recipe-card ">
+         <article class="all-card-icons">
+           <img class="plus-icon card-icon" src="./images/plus-icon.png" alt="plus icon used to expand and show recipe details">
+           <img class="star-icon card-icon" src=${src}
+          "alt="empty star icon used to favorite and unfavorite recipes">
+         </article>
+         <article class="card-image-section">
+           <img class="card-image" src="${recipe.image}" alt="sample display of recipe">
+         </article>
+         <article class="recipe-name-area">
+           <h2 class="recipe-name">${recipe.name}</h2>
+         </article>
+       </article>`
       })
-    })
-    .catch(err => {
-      console.log(err);
-      alert('Sorry, the recipes failed to load. Try again later.')})
-  },
 
-  grabUsers() {
-    return fetch('https://fe-apps.herokuapp.com/api/v1/whats-cookin/1911/users/wcUsersData',)
-    .then(response => {
-      return response.json()
-    })
-    .then(data => {
-      console.log(domUpdates.randomNumber)
-      greetUser(data.wcUsersData[domUpdates.randomNumber -1])
-       return data.wcUsersData;
-        })
-    .catch(err => {
-        console.log(err);
-        alert('Sorry, the user information failed to load. Try again later.');
-    })
   },
-
-    grabIngredients() {
-    return fetch('https://fe-apps.herokuapp.com/api/v1/whats-cookin/1911/ingredients/ingredientsData')
-    .then(response => {
-        return response.json()
-    }).then(ingredientData => {
-        return ingredientData.ingredientsData;
-        })
-    .catch(err => {
-        console.log(err);
-        alert('Sorry, the ingredients failed to load. Try again later.');    
-  })
- },
-mergeFetchTimelines() {
```

Comment:
I think it was a good idea to move the fetch requests away from `domUpdates` since they aren't technically responsible for updating the DOM.  This is something we can ask for clarification though!